apps/frontend/src/components/global/Sidebar.vue

```diff
@@ -91,7 +91,15 @@
           <!-- Filtering Capabilities Section -->
           <QuickFilters />
           <v-divider class="my-5" />
-          <CategoryFilters />
+          <DropdownFilters
```

Comment:
i think we should try to set up an hour with a uiux person to figure out a good path forward for the sidebar stuff

context to explain:
- searchbar at the top
- clickable ui in the main page that lets a user filter based on whacking a pie chart
- adding in new ui to the pull out bar that lets a person do filtering more easily there
- sidebar area already had 'filtering' in place by being able to select/deselect files

questions to ask:
- file metadata / filename specific searchbar in the sidebar? or a full copy of the primary search bar? or nothing at all?
- splitting ui areas between file metadata search stuff and infile search stuff (ex. metadata stuff at the top, infile at the bottom, file de/select in the middle OR both filtering stuff at the bottom and file de/select stuff at the top
- what order to put everything in in the sidebar
- is the expansion panel concept to hide files fine?
- is transitioning pages via clicking the expansion panels fine?

Reply:
talked at length with eugene/aaron

turns out that we don't need to have a searchbar in the sidebar, so the argument for having that complete separation of the filtering areas goes away since there isn't gonna be a separate area in the first place.  consequently, the current positioning of the file filters thing is fine.

we also got consensus to be able to do something (as of yet undetermined) to mark files as being hidden.  unsure if that's make a separate 'hidden' drop down thing, or just do a strikethrough of the name, or whatever.  regardless, make it happen in the sidebar and potentially the carousel as well.